staging-process.md

```diff
@@ -0,0 +1,17 @@
+# Staging Process Proposal
+
+Anyone may submit an idea for a policy or program following the staging process outlined below. To submit an idea, create a proposal document in markdown format in the /proposals directory, submit a pull-request, and apply the appropriate staging label. 
+
```

Comment:
Should we ask that the stage be in the title or have the markdown files live in subdirectories for example: /proposals/stage-0